httpserver/main.cpp

```diff
@@ -63,33 +64,57 @@ bool parseLogLevel(const std::string& input,
     return false;
 }
 
+
+static boost::optional<boost::filesystem::path> get_home_dir() {
+
+    /// TODO: support default dir for Windows
+#ifdef WIN32
+    return boost::none;
+#endif
+
+    char* pszHome = getenv("HOME");
+    if (pszHome == NULL || strlen(pszHome) == 0)
+        return boost::none;
+
+    return boost::filesystem::path(pszHome);
+}
+
 int main(int argc, char* argv[]) {
     try {
         // Check command line arguments.
-        if (argc < 2) {
-            usage(argv);
-            return EXIT_FAILURE;
-        }
-
         std::string lokid_key_path;
-        std::string db_location(".");
+
+        namespace fs = boost::filesystem;
+
+        const auto home_path = get_home_dir();
+        const fs::path storage_path = home_path ? (*home_path / ".loki" / "storage") : ".";
+
+        std::string db_location = storage_path.string();
         std::string log_location;
         std::string log_level_string("info");
         bool print_version = false;
         uint16_t lokid_rpc_port = 22023;
 
         po::options_description desc;
-        desc.add_options()("lokid-key", po::value(&lokid_key_path),
-                           "")("db-location", po::value(&db_location),
-                               "")("output-log", po::value(&log_location), "")(
-            "log-level", po::value(&log_level_string),
-            "")("version,v", po::bool_switch(&print_version),
-                "")("lokid-rpc-port", po::value(&lokid_rpc_port));
+        // clang-format off
+        desc.add_options()
+            ("lokid-key", po::value(&lokid_key_path), "Path to the Service Node key file")
+            ("db-location", po::value(&db_location),"Path to the Storage Server database location")
+            ("output-log", po::value(&log_location), "Path to the log file")
```

Comment:
We should also store the logfile in the base path as well further down?
```
if (vm.count("output-log")) {
.....
  log_location += ".out";
.....
  auto sink = logging::add_file_log(log_location); // put base path/log_location?
```

Reply:
OK, but we might do that in a separate PR.